pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -306,6 +312,7 @@ type CertificateSpec struct {
 	// Defines extra output formats of the private key and signed certificate chain
 	// to be written to this Certificate's target Secret.
 	// +optional
+	// +listType=atomic
 	AdditionalOutputFormats []CertificateAdditionalOutputFormat `json:"additionalOutputFormats,omitempty"`
```

Comment:
Could this be a map and use the `Type` as the key?

Reply:
Yes, it can! Done!